File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
import java.util.*;

class Solution {
    public int[] solution(String my_string) {
        Map<Character, Integer> abc = new HashMap<>();
        char upper = 'A';
        char lower = 'a';
        
        for (int i = 0; i <= 25; i++) {
            abc.put((char)(upper + i), 0);
            abc.put((char)(lower + i), 0);
        }
        
        for (char letter : my_string.toCharArray()) {
            if (abc.containsKey(letter)) {
                abc.put(letter, abc.get(letter) + 1);
            }
        }

        int[] answer = new int[52];
        int index = 0;
        for (int value : abc.values()) {
            answer[index++] = value;
        }

        return answer;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    frequency = [0] * 52
    
    for letter in my_string:
        if 'A' <= letter <= 'Z':
            frequency[ord(letter) - ord('A')] += 1
        elif 'a' <= letter <= 'z':
            frequency[ord(letter) - ord('a') + 26] += 1
    
    return frequency